### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by Jaxelr** [uglybugger/NotDeadYet#17]
Ive been using this project for a few years, but i barely see activity nowadays. Is it still Alive?

**Comment by uglybugger** [comment]
Yes, it’s still a thing :) I wouldn’t mind a hand with its maintenance, though, as I’m pretty stretched.

**Comment by Jamleck** [comment]
Hi Andrew, I'd like to help with the maintenance of this project. Let me know what needs to be done :) I'll start by trying to understand the codebase, release and versioning process.

**Comment by uglybugger** [comment]
I’d love some help :)

Current plans are to move it to .NET Core and get it building on Appveyor. You’re welcome to take a look if you like.

**Comment by Jamleck** [comment]
@uglybugger  I'll move it to Net Core 2.0 and get it building on AppVeyor.

- [x] Create .Net Core 2.0 port . It seems like some work was done in [PR13](https://github.com/uglybugger/NotDeadYet/pull/13). I notice that there is a dependency on ThirdDrawer.* which needs to be updated as well.
- [x] Create a sample
- [x] Move build and packaging process to Appveyor

As a separate issue, I'd like to add support for ASP.NET Core using the default dependency injection mechanism as described [here](https://lostechies.com/jimmybogard/2016/07/20/integrating-automapper-with-asp-net-core-di/
).
What are your thoughts?

**Comment by Jaxelr** [comment]
Can we close this issue an open a new one Issue for the migration to Core? i just don't want this information buried inside an issue which was mostly a question.

**Comment by flytzen** [comment]
FYI: #13 did finish the work, bar NUnit, and did add a sample. It also removed the dependency on ThirdDrawer :)

**Comment by Jamleck** [comment]
@flytzen I had a look at PR https://github.com/uglybugger/NotDeadYet/pull/13 and was able to use your work. I went down the route of implementing AspNetCore support using AspNetCore middleware and dependency injection. I also moved the NuSpec files into the new .csproj format.

I've migrated the UnitTest project to the new .csproj format and I've gotten the [AppVeyor](https://ci.appveyor.com/project/Jamleck/notdeadyet) build and versioning working. ~~I'm just testing publishing the generated Nuget packages to MyGet then I'll issue a new pull request.~~ See PR https://github.com/uglybugger/NotDeadYet/pull/20
